### DISCUSSION TITLE
Release 0.5.2

### DISCUSSION THREAD

**Discussion by lucky6qi** [lmcinnes/umap#782] [General]
Hi umap team, when do you expect that the version 0.5.2 can be released on pip, so that we could use the newest  one!

**Comment by lmcinnes** [comment]
Soon? I think? I don't actually have a timeline on that unfortunately.

**Reply by lucky6qi** [reply]
Thanks for fast reply. soon means this month or depend on other things?

**Comment by lmcinnes** [comment]
Likely this month, possibly the next. Was there a particular bug fix or
feature that you were wanting?

**Reply by lucky6qi** [reply]
Fixes #741 I would like to have.
